Fixing query documentation and examples

mikibonacci opened the pull request:
This fixes https://github.com/aiidateam/aiida-atomistic/issues/47
This fixes https://github.com/aiidateam/aiida-atomistic/issues/51

mikibonacci committed:
Default values of variables is not stored in db

we avoid unnecessary or redundant storing.

unkcpz commented on conftest.py:
```diff
@@ -48,7 +48,7 @@ def example_structure_dict():
                 "masses": 63.546,
                 "charges": 1.0,
-                "magmoms": [0.0,0.0,0.0],
+                #"magmoms": [0.0,0.0,0.0],
```
If this is not used, maybe remove it??

mikibonacci replied:
right, will do it thanks!!

unkcpz commented on src/aiida_atomistic/data/structure/site.py:
```diff
@@ -188,14 +195,14 @@ def atom_to_site(
             masses = _atomic_masses[symbols] if masses is None else masses
             weights = _default_values["weights"] if weights is None else weights
```
Is this `_default_values` should be the `_DEFAULT_VALUES`?

mikibonacci committed:
Moving default into init of data.structure

and some small fixing requested in the PR.

mikibonacci committed:
Adding two computed fields

- is_alloy
- has_vacancies

This will be useful in queries.

mikibonacci committed:
Providing documentation on Query

This fixes almost the whole https://github.com/aiidateam/aiida-atomistic/issues/47

mikibonacci committed:
cell_charge and cell_magmom.

mikibonacci committed:
Removing small bug
